courses/free-courses-en.md

```diff
@@ -1111,6 +1111,7 @@
 * [Python Programming Essentials](https://www.coursera.org/learn/python-programming) - Scott Rixner, Joe Warren (Coursera)
 * [Python Programming From Scratch With Practicals](https://www.tutorialspoint.com/python_programming_from_scratch_with_practicals/index.asp) - Sundeep Saradhi Kanthety (Tutorials Point)
 * [Python Programming Language](https://www.geeksforgeeks.org/python-programming-language/) (Geeks for Geeks)
+* [Python Programming MOOC 2022](https://programming-22.mooc.fi) - University of Helsinki
```

Comment:
Looks like this should be in interactive tutorials isn't it @SimonLariz.

Reply:
Also we might need to change the affiliation to  Agile Education Research group of University of Helsinki

Reply:
we'll take it as a course.